src/server/object_services/map_server.js

```diff
@@ -417,16 +417,20 @@ async function make_room_in_tier(tier_id, bucket_id) {
 
         const tiering_status = node_allocator.get_tiering_status(tiering);
         const next_tier = mapper.select_tier_for_write(tiering, tiering_status, tier_and_order.order + 1);
-        if (!next_tier) {
+
+        if (!next_tier && !bucket.namespace.caching.eviction) {
```

Comment:
I don't really get this... seems like we must have eviction with caching - there is a default always (correct me if wrong) - so shouldn't we check just for namespace? or to add a no-eviction option to this PR?

Reply:
I added this here as at the moment, we have just one eviction policy, LRU. In future if we have any other policy, it will be very easy to update the code. If eviction policy is not defined, we let the default behavior go through.
I have updated all other issues

Reply:
irrelevant in latest changes